third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning-expected.txt

```diff
@@ -11,5 +11,6 @@ Tests that Lighthouse panel displays a warning when important data may affect pe
 [x] Clear storage
 [x] Simulated throttling
 Generate report: enabled visible
-Warning Text: There may be stored data affecting loading performance in these locations: Web SQL, IndexedDB. Audit this page in an incognito window to prevent those resources from affecting your scores.
+PROMISE FAILURE: TypeError: Lighthouse.LighthousePanel.getEvents is not a function
```

Comment:
https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/lighthouse/LighthousePanel.js;l=118;drc=068400262b841bb557d3322f296406a6272be029

hmm

Reply:
oh its cuz devtools checkout is old. sending a PR to fix that.